Tests/FluentAssertions.Specs/Equivalency/BasicEquivalencySpecs.cs

```diff
@@ -596,6 +596,56 @@ public void When_the_graph_contains_guids_it_should_properly_format_them()
             act.Should().Throw<XunitException>().WithMessage("Expected item[0].Id to be *-*, but found *-*");
         }
 
+        [Fact]
+        public void When_defining_comparing_by_members_with_callback_that_returns_false_then_the_normal_equality_strategy_process_should_be_used()
```

Comment:
🙃 I'm nitpicking here, but we prefer to keep those test names functional and never refer to the literal name of an identifier, e.g. `When_none_of_the_members_should_be_compared_by_members_it_should_succeed`